politeiawww/api/v1/v1.go

```diff
@@ -105,6 +105,7 @@ const (
 	ErrorStatusInvalidSigningKey           ErrorStatusT = 25
 	ErrorStatusCommentLengthExceededPolicy ErrorStatusT = 26
 	ErrorStatusUserNotFound                ErrorStatusT = 27
+	ErrorStatusUserNotPaidPaywall          ErrorStatusT = 28
```

Comment:
Let's just call the error `ErrorStatusUserNotPaid` or `ErrorStatusUserPaywall`.

Reply:
fixed